lib/Choices.js

```diff
@@ -17,7 +17,7 @@ module.exports = class Choices {
    * Create a set of Scenarios and Choices to make.
    * @param {ChoicesConfig} config defining scenarios, socket, and say
    */
-  constructor({scenarios, socket, say}) {
+  constructor({scenarios, socket, say, args}) {
```

Comment:
This requires some small changes in other spots too, but since args would be best as an arbitrary object, I'd suggest defaulting it to `{}`

This could be reflected in the documentation, and then it would be passed into the API as `{startingAttributes}` for the example.